extensions/resource-deployment/src/ui/pageLessDeploymentModel.ts

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { DeploymentProvider, instanceOfCommandDeploymentProvider, instanceOfDialogDeploymentProvider, instanceOfDownloadDeploymentProvider, instanceOfNotebookDeploymentProvider, instanceOfWebPageDeploymentProvider } from '../interfaces';
+import { DeploymentInputDialog } from './deploymentInputDialog';
+import { ResourceTypeModel } from './resourceTypeModel';
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as nls from 'vscode-nls';
+import { ResourceTypePage } from './resourceTypePage';
+const localize = nls.loadMessageBundle();
+
+
+export class PageLessDeploymentModel extends ResourceTypeModel {
+
+	initialize(): void {
+		const pages: ResourceTypePage[] = [];
```

Comment:
nit suggestion: changing to just makes the code more concise and improves readability:
this.wizard.setPages([]);